Source/Views/HeaderView.swift

```diff
@@ -12,6 +12,9 @@ open class HeaderView: UIView {
       attributes: LightboxConfig.CloseButton.textAttributes)
 
     let button = UIButton(type: .system)
+    button.backgroundColor = UIColor.black.withAlphaComponent(0.2)
+    button.layer.cornerRadius = 10
```

Comment:
nit: 8px corner radius in both places

Reply:
SGTM. I just did what ChatGPT suggested. 😬